compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/scene/BaseComposeScene.skiko.kt

```diff
@@ -205,6 +206,7 @@ internal abstract class BaseComposeScene(
             nativeEvent = nativeEvent,
             button = button
         )
+        recomposer.performScheduledEffects()
```

Comment:
We have a failed test `catch only the first scroll event in one frame`.

We can remove it, as this behaviour is no longer needed and it is written based on maybe incorrect interpretation of `pointerInput` + `awaitPointerEventScope`.

A context why it was written:

In Jetpack Compose 1.0 we had very slow handling of a scroll event. Native systems on the other hand can send 20 events per frame, and it is much slower to handle 20 events instead of one. So we decided to handle only the first one, relying on coroutine dispatching inside `pointerEvent`.

Now we don't have such issues, I checked a heavy Lazy List, there are no difference in FPS/lag comparing to `jb-main`. If we decide in the future that we need some event filtering/batching, we should do it differently, not relying on `pointerInput` dispatching.

Reply:
I removed it, it needs to be merged for 1.7.0-beta01